### DISCUSSION TITLE
Migration to 27.0.2

### DISCUSSION THREAD

**Discussion by elephantastyczny** [nextcloud/all-in-one#3203] [Questions] [answered]
Hi All,

I just wanted to ask you if some of you already migrated to NC27?
Is everything working after migration?
Is it faster/slower/same?

Is there any component / plugin that is confirmed not working?

**Comment by szaimen** [comment, marked as answer]
Hi, this is probably better asked at https://help.nextcloud.com

**Comment by elephantastyczny** [comment]
Ok, thanks!

W dniu pt., 18.08.2023 o 22:28 Simon L. ***@***.***>
napisał(a):
